src/main/scala/chiseltest/package.scala

```diff
@@ -143,6 +196,11 @@ package object chiseltest {
         }.toSeq
         chiselTypeOf(x).Lit(elementValueFns: _*).asInstanceOf[T]
       }
+      case (x: Vec[_]) =>
+        val elementValueFns = x.getElements.zipWithIndex.map { case (elt: Data, index: Int) =>
```

Comment:
Does this discard the index? At least, do you want to add an assertion, if not propagate the index through?

Reply:
Removed unneeded `zipWithIndex`